clusterloader2/cmd/clusterloader.go

```diff
@@ -100,6 +101,27 @@ func completeConfig(f *framework.Framework) error {
 	return nil
 }
 
+func printClusterNodes(f *framework.Framework) error {
```

Comment:
Can we move it out of main file? Probably to some utility file somewhere (in framework?)